success-criteria.md

```diff
@@ -501,8 +473,13 @@ stakeholder group.
     open marketplace of vendors.
 
 -   Appropriate risk mitigation and remedies:
```

Comment:
Not enough under this section. As a publisher we have a number of additional concerns that we’d like to see here. Proposed additions follow.